tox.ini

```diff
@@ -36,3 +36,41 @@ deps =
 commands =
     pip wheel -w {envtmpdir}/build --no-deps .
     twine check {envtmpdir}/build/*
+
+[testenv:black]
+skip_install = True
+deps = black
+commands = black .
```

Comment:
no diff command?

Reply:
This one is meant for when you want to run black on your code - in the linting step we use --diff